src/components/graph/graph.ts

```diff
@@ -407,6 +412,24 @@ export default class GraphComponent {
       })
       .attr('fill', 'none')
       .attr('stroke', GraphComponent._LINK_STROKE_COLOR)
+      .attr('stroke-width', '6px')
+      // Jump to other side of the link
+      .on('click', (d: ILinkTuple) => {
```

Comment:
This .on click could be moved to its on method, but if u want to leave it here thats fine too